### PR TITLE
add description for completion command

### PR DESCRIPTION
**Pull request by valentyn88**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by FriggaHel — APPROVED** [review]

File: docs/testrunner-toolkit/saucectl.md [hunk 1]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
...
+saucectl completion powershell | Out-String | Invoke-Expression
+```
+---
+
+To load completions for every new session, run:
+
+```bash
+saucectl completion powershell > saucectl.ps1
+```
+---
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/saucectl.md:484]:
```suggestion
```

File: docs/testrunner-toolkit/saucectl.md [hunk 2]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
...
+
+and source this file from your PowerShell profile.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/saucectl.md:486]:
```suggestion
```
I am afraid this last statement might get lost under the example. Have I written it to make sense above?

File: docs/testrunner-toolkit/saucectl.md [hunk 3]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
...
+
+```bash
+saucectl completion bash > /usr/local/etc/bash_completion.d/saucectl
+```
+---
+
+### zsh
+
+If shell completion is not already enabled in your environment, you will need to enable it.
+You can execute the following once:
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

File: docs/testrunner-toolkit/saucectl.md [hunk 4]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
...
+
+```bash
+$ echo "autoload -U compinit; compinit" >> ~/.zshrc
+```
+---
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

File: docs/testrunner-toolkit/saucectl.md [hunk 5]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
...
+  
+To load completions for each session, execute once:
+
+```bash
+saucectl completion zsh > "${fpath[1]}/_saucectl"
+```
+---
+
+You will need to start a new shell for this setup to take effect.
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

File: docs/testrunner-toolkit/saucectl.md [hunk 6]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
...
+saucectl completion fish > ~/.config/fish/completions/saucectl.fish
+```
+---
+
+### PowerShell
+
+```bash
+saucectl completion powershell | Out-String | Invoke-Expression
+```
+---
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

File: docs/testrunner-toolkit/saucectl.md [hunk 7]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
...
+
+To load completions for every new session, run:
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

File: docs/testrunner-toolkit/saucectl.md (hunk 2 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

File: docs/testrunner-toolkit/saucectl.md (hunk 1 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

File: docs/testrunner-toolkit/saucectl.md [hunk 8]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
...
+---
+
+You will need to start a new shell for this setup to take effect.
+
+### fish
+
+```bash
+saucectl completion fish | source
+```
+---
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

File: docs/testrunner-toolkit/saucectl.md [hunk 9]
```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
 ```bash
 saucectl run --verbose
 ```
+
+## `$ saucectl completion`
+
+Allows you to generate completion script for `bash`, `zsh`, `fish` and `powershell` shells.
```
[the remark this answers is not in the document]
Comment (valentyn88) [reply docs/testrunner-toolkit/saucectl.md]: I fixed this issue.

**Review by sweeneyskirt-sl — APPROVED** [review]
LGTM

**Pull request merged**